driver/terraform.go

```diff
@@ -547,6 +541,26 @@ func (tf *Terraform) initTaskTemplate() error {
 	return nil
 }
 
+func (tf *Terraform) setNotifier(tmpl templates.Template, serviceCount int) {
+	switch tf.task.Condition().(type) {
+	case *config.CatalogServicesConditionConfig:
+		tf.template = notifier.NewCatalogServicesRegistration(tmpl, serviceCount)
+	case *config.ConsulKVConditionConfig:
+		tf.template = notifier.NewConsulKV(tmpl, serviceCount)
+	case *config.ScheduleConditionConfig:
+		// If a ConsulKVSourceInputConfig is specified, then we need to add
+		// to the number of services passed to the notifier, since consul-kv adds a dependency
+		serviceCountAddition := 0
```

Comment:
This isn't really a services count at this point, it's more a dependencies count. I suggest changing the name of this variable here, and also in the `NewSuppressNotification` method so it's more accurate.